migrations/2024.11.08T12.51.32.bundles.root_transaction_id-index.sql

```diff
@@ -0,0 +1 @@
+CREATE INDEX IF NOT EXISTS bundle_data_items_root_transaction_id_id_idx ON bundle_data_items (root_transaction_id, id);
```

Comment:
I'm not sure about this. The advantage of having `id` in the index is that SQLite won't have to pull the `id` from the row. The disadvantage is that it makes the index larger. The `root_transaction_id` should be enough for fast lookup either way.